napalm_sros/sros.py

```diff
@@ -1170,23 +1180,36 @@ def _update_buff(buff):
                     return new_buff[: new_buff.rfind("\n")]
 
                 if retrieve == "running":
-                    buff_running = self._perform_cli_commands([cmd_running], True)
-                    configuration["running"] = _update_buff(buff_running)
+                    buff_running = self._perform_cli_commands(cmd_running, True)
+                    configuration["running"] = _update_buff(buff_running, skip_persistent_indices=True)
+                    if self.retrieve_bof_config:
+                        buff_bof = self._perform_cli_commands(cmd_bof_running, True)
+                        configuration["running"] += f"\n{_update_buff(buff_bof, skip_persistent_indices=False)}"
```

Comment:
I would prefer to use
```
configuration["bof"]
```
here, to not mix different sections of the config. They are separate for a reason

Reply:
Yes, I definitely get your point. I opted for go for this route as it suits our use case best. I could add another parameter to choose where to add the `bof` configuration.

Please let me know what you think, if you still think it is best to just `configuration["bof"]` at all times that is fine by me. I'll use a custom version of the driver in our environment in that case.